Lib/subprocess.py

```diff
@@ -721,12 +721,20 @@ def __init__(self, args, bufsize=-1, executable=None,
 
         self._closed_child_pipe_fds = False
 
+        if self.text_mode:
+            if bufsize == 1:
+                line_buffering = True
+                # line buffering is not supported in binary mode
```

Comment:
But here we are in text mode, no? The comment is misleading, I suggest to remove it.

Or say something like: "# Use the default buffer size for the binary buffered reader/writer".

Reply:
I was referring to the binary mode used for the underlying streams, and I didn't realize that it can be misleading, but now I see it. Changed to clearer wording, thanks!